tachyon/zk/plonk/circuit/region.h

```diff
@@ -98,6 +98,91 @@ class Region {
       return Error::kNone;
     }
   };
+
+  explicit Region(Layouter* layouter) : layouter_(layouter) {}
+
+  // See the comment above.
+  Error EnableSelector(AnnotateCallback annotate, const Selector& selector,
+                       size_t offset) {
```

Comment:
```suggestion
                       size_t offset) override {
```
Plus, if we get to add override keyword, how about removing " // See the comment above." comment?